src/perf/tmo-wrapper.js

```diff
@@ -0,0 +1,129 @@
+// Module with partial logic from
+// https://github.com/mozilla/telemetry-dashboard/blob/gh-pages/wrapper/telemetry-wrapper.js
+// This module focuses on data manipulations rather than data plotting
+import Telemetry from './telemetry-node';
+import { getEvolution } from './tmo';
+import getVersions from '../release/versions';
+import TELEMETRY_CONFIG from './telemetry_config';
+
+// This function is a copy/paste from telemetry-wrapper
+const createTMOLinkForParams = (params) => {
+  const url = `https://telemetry.mozilla.org/new-pipeline/${params.evoVersions ? 'evo' : 'dist'}.html#!`;
+
+  const queryParams = [];
```

Comment:
Replace this whole block of array concatenation with an object using `qs.stringify` (example [here](https://github.com/mozilla/platform-health/blob/08b87f1983c1e808dbad5905795ad728606d198f/static/quantum/perfherder.js#L30))